framework/resources/Datasources/table_function/DRILL-5166_generate_data.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
Do we generate data every time we run automation? How big is the dataset and how long it takes to create the dataset? We want to reduce time spend in datagen phase. Please look into if we can generate data once and reuse in subsequent runs. Thanks.

Reply:
I agree. Please share how big the dataset is - depending on it, we could check it in to the repo or upload to S3.

Reply:
We can reuse it, good point.
Regarding to the size, I need the file larger than the maprfs chunk size, wich is 256MB by default. However I noticed that the Test Framework creates directories with chunk size 64MB and therefore I can reduce the file size to ~70MB.

Reply:
Cool, then lets upload it to S3 (if you do not have an account, share the data over OneDrive and I'll upload it). Once it's available, i'll share the URL and you can add it to pom.xml, and update the tests to use the downloaded data instead. We'll no longer need this script then.